Mathlib/Data/Set/Function.lean

```diff
@@ -688,6 +688,10 @@ lemma exists_eq_graphOn [Nonempty β] {s : Set (α × β)} :
   .trans ⟨fun ⟨f, t, hs⟩ ↦ ⟨f, by rw [hs, image_fst_graphOn]⟩, fun ⟨f, hf⟩ ↦ ⟨f, _, hf⟩⟩
     exists_eq_graphOn_image_fst
 
+lemma graphOn_prod_graphOn (s : Set α) (t : Set β) (f : α → γ) (g : β → δ) :
+    s.graphOn f ×ˢ t.graphOn g = Equiv.prodProdProdComm .. ⁻¹' (s ×ˢ t).graphOn (Prod.map f g) := by
```

Comment:
Is there a reason to prefer the preimage over the image spelling on the rhs?

Reply:
Preimages along equivs are usually preferred to images due to better defeqs